packages/rfw/example/wasm/macos/Runner/DebugProfile.entitlements

```diff
@@ -8,5 +8,7 @@
 	<true/>
 	<key>com.apple.security.network.server</key>
 	<true/>
+	<key>com.apple.security.network.client</key>
+	<true/>
```

Comment:
should we add this to the other entitlements too? (notably the release mode one)

Reply:
This should definitely be in the release entitlements as well.